### ISSUE TITLE
Disable SideNav for user roles

### ISSUE THREAD

**Issue by pgrill79** [dashjoin/platform#356]
In Dashjoin 3.X it was possible to hide the Sidenave completely, now you can only set the properties:

"sidenav-open": "false" und  "sidenav-width-px": "0" 

but the Menu Button remains and only circles if you press it (and the text between open / close menu changes)

**Comment by pgrill79** [comment]
Haven't found a setting for this yet - can you give me a hint ? ;)

**Comment by aeberhart** [comment]
You need to logout and back in. All UI config settings are transferred only when loggin in.

**Comment by pgrill79** [comment]
I tried this (also cleared the cache), but no effect ... the Button on the top left is still there.

**Comment by aeberhart** [comment]
strange... just tested locally with different users and sidenav settings.
did you also reload the browser to make sure the new single page javascript code is loaded?

**Comment by pgrill79** [comment]
I just killed all chrome and edge processes and also cleard the cache, but nevertheless i see the button (in both browsers):
![image](https://github.com/dashjoin/platform/assets/47058152/d5f8025d-d788-4c39-8d88-f9a95f9144c5)

These are the settings i have defined in the role of the user:
![image](https://github.com/dashjoin/platform/assets/47058152/297513da-ef3e-4351-b5b1-074baf84df51)

**Comment by aeberhart** [comment]
ok - if the properties are used, the settings are stored as strings. Thus 

```
    const sideNavWidth = profile.getVariable()['sidenav-width-px']
    const sx = sideNavWidth === 0 ? {
```

treats sidenavwidth as != 0

**Comment by aeberhart** [comment]
fixed in 5.2
backend casts map values for boolean / integer profile settings
